src/wasm2js.h

```diff
@@ -80,6 +80,17 @@ void sequenceAppend(Ref& ast, Ref extra) {
   ast = ValueBuilder::makeSeq(ast, extra);
 }
 
+bool isTableExported(Module& wasm) {
+  if (!wasm.table.exists || wasm.table.imported())
+    return false;
+  for (auto& ex : wasm.exports) {
+    if (ex->value == wasm.table.name) {
```

Comment:
this should also check `ex->kind == ExternalKind::Table` as the name is unique only for each Kind.

Reply:
Oh!  I didn't know that.  Thanks for pointing that out.